cachet-tools/hosted-services.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
This is fine to have in here for now, but down the road I really want this to be fetched from app-interface.

Reply:
sure when it becomes available. If already available, where to start?